resources/github-comment-markdown.template

```diff
@@ -113,11 +113,10 @@ ${description}
   <%}%>
   </p>
   </details>
-<%} else if (!buildStatus?.equals('ABORTED')) {%>
+<%} else if (!buildStatus?.equals('ABORTED') && testsSummary?.failed == 0) {%>
```

Comment:
If there are test errors, shouldn't it be:
```suggestion
<%} else if (!buildStatus?.equals('ABORTED') && testsSummary?.failed > 0) {%>
```

Reply:
Use Case:

- If there are test failures then do no report a syntax error regardless of the build status.
- If no test errors and build status different than `aborted` then report a syntax error

I guess `(!buildStatus?.equals('ABORTED') && testsSummary?.failed == 0) ` is the one supporting the above use cases, right?

Reply:
Ahh thanks for the clarification. Reading the title made me thought that way